sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -152,6 +152,37 @@ def _set_rewrite_response_callback(self, callback):
     """
     self._rewrite_cb = callback
 
+  def get_bucket(self, bucket_name):
+    """Returns an object bucket from its name, or None if it does not exist."""
+    try:
+      request = storage.StorageBucketsGetRequest(bucket=bucket_name)
+      return self.client.buckets.Get(request)
+    except HttpError:
+      return None
+
+  def insert_bucket(self,
+                    bucket_name,
+                    project,
+                    kms_key=None,
+                    location=None):
+    """Create and return a GCS bucket in a specific project."""
```

Comment:
Please document return types and what None means. (or better yet don't return None on failure; just pass the exception)